### PR TITLE
Update colors

### LINKED ISSUE #6: Add symbolic links to generated figures
Fix `TODO` in [task_create_screencasts.py](../blob/main/src/ada_topics/task_create_screencasts.py).

### PR DESCRIPTION
**Pull request by timmens**
In this PR, I

- Improve the tables on the website
- Improve the tables on the slides in light and dark mode
- Improve the background of the quiz in dark mode on the website
- Make the background of the code blocks on the slides invisible for printing 
- Update all tables used in the screencasts (manually!)

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (timmens) [inline .pre-commit-config.yaml:1]: Some fixes such that hooks don't complain. Changes from #7 will overwrite it.

**Review by hmgaudecker — COMMENTED** [review]
Looks good to me, but does not quite work yet as expected. I changed a few things trying to fix that (pls. check the last commit in particular in that respect), but to no avail.

In particular, `npx slidev src/ada_topics/pandas_data/datatypes/screencast/slides.md` shows me this on slide 3:

image

If not too difficult, could we change the background of code in the dark theme to be like 95% black and 5% the ada-blue? *(happy to do fine-tuning myself once I'd know where to do so)*

image

File: src/ada_topics/slidev_config/style.css [hunk 1]
```diff
@@ -51,8 +51,13 @@ td {
   background-color: #152b41 !important;
 }
 
+.dark .slidev-code {
+  background-color: rgba(0, 0, 0, 0.25) !important;
```
Comment (timmens) [inline src/ada_topics/slidev_config/style.css:55]: If the last number is 0, the background of code blocks will be transparent. With 1 it is black.

**Review by hmgaudecker — APPROVED** [review]
Thanks a lot!

**Pull request merged**